Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -321,6 +331,27 @@ - (void)hideBanner
 #endif
 }
 
+- (void)_minimumHideDelayReached
+{
+    _blockHideBanner = NO;
+    if (_hideBannerAfterMinimumHideDelayReached) {
+        [self hideBanner];
+        _hideBannerAfterMinimumHideDelayReached = NO;
+    }
+}
+
+- (void)_bannerDismissalRecognized:(NSNotification*)notification
```

Comment:
Can't the method declaration also be behind `#if ENABLE(FULLSCREEN_DISMISSAL_GESTURES)`?